virtual_ecosystem/models/animal/animal_communities.py

```diff
@@ -41,8 +42,8 @@ class AnimalCommunity:
         data: The core data object
         community_key: The integer key of the cell id for this community
         neighbouring_keys: A list of cell id keys for neighbouring communities
-        get_destination: A function to return a destination AnimalCommunity for
-            migration.
+        get_community_by_key: A function to return a designated AnimalCommunity by
+        integer key.
```

Comment:
"integer key" needs to be indented with 4 spaces here, otherwise `mkdocstrings` looks for an argument called "integer"